docs/_documentations/codewind-openapi-eclipse.md

```diff
@@ -18,13 +18,17 @@ The Codewind OpenAPI Tools for Eclipse includes wizards that invoke the OpenAPI
 2. [Optional] Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
 3. Install the [Codewind OpenAPI Tools from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
 
-## Running commands
+## Running commands, generating, and building client and server stubs by using the OpenAPI tools
```

Comment:
Step 4 mentions HTML docs too.  I'd remove `Running commands`.   How about:

Generating HTML, and client and server stubs by using the OpenAPI wizard